frontend/views/containers/user-settings/AppLogs.vue

```diff
@@ -0,0 +1,147 @@
+<template lang='pug'>
+  .settings-container
+    section.card
+      h2.is-title-3.c-title Application logs
+
+      .c-header
+        fieldset.c-filters
+          legend.sr-only Visible logs
+          label.checkbox
+            input.input(type='checkbox' name='filter' v-model='filter' value='error')
+            i18n Errors
+          label.checkbox
+            input.input(type='checkbox' name='filter' v-model='filter' value='warn')
+            i18n Warnings
+          label.checkbox
+            input.input(type='checkbox' name='filter' v-model='filter' value='debug')
+            i18n Debug
+
+        button.is-small.c-download(@click='downloadLogs')
+          i.icon-download.is-prefix.c-icon
+          i18n Download
+          //- i18n.hide-touch Download
+          //- i18n.hide-desktop Share
+        a(ref='linkDownload' hidden)
+
+      textarea.textarea.c-logs(ref='textarea' rows='12' readonly)
+        | {{ prettyLog }}
+
+      .buttons
+        i18n.link(tag='a' @click='openTroubleshooting') Troubleshooting
+        button.is-small.is-outlined(@click='clearLogs') Clear Logs
```

Comment:
@mmbotelho,  @pieer and @taoeffect, what do you think of giving this possibility to the user?

Reply:
I feel it's very advanced user action. I'm not sure why he would need to clear the logs other than someone being concern about memory management. But I think the log will have a max size anyway, isn't it?

Reply:
Agreed, I just wanted to double check. I'll make sure to remove this before merging the code. (it's useful to review/debug).

Reply:
I agree with @pieer !

Reply:
Cool. I'll keep it here until the CR is done. Before merging it, I'll delete it.